charts/victoria-metrics-cluster/templates/_helpers.tpl

```diff
@@ -6,13 +6,42 @@ Expand the name of the chart.
 {{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{/*
```

Comment:
please bump version of Chart in `Chart.yaml`

Reply:
Ofc! My bad!